keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -808,7 +808,8 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Conformance tests for the fields in `/metrics/cadvisor` should be created.
 - Validate performance impact of this feature is within allowable margin (or non-existent, ideally).
 	- The CRI stats implementation should perform better than they did with CRI+cAdvisor.
-- cAdvisor stats provider may be marked as deprecated (depending on stability of new CRI based implementations).
+- cAdvisor stats provider will be marked as deprecated.
```

Comment:
To be clear, is this referring to the cAdvisor stats provider which _only_ surfaces metrics from cAdvisor and not CRI? The cri stats provider now has two paths, combine cri stats + cAdvisor stats and the strict CRI stats you added. 

I assume we are talking about the first category, i.e. when CRI stats are not used at all. Is it correct?

It will also require CRI-O to stop usage of it :) 

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cadvisor/util.go#L71-L73

Reply:
this, and also cadvisor providing `/metrics/cadvisor` which maybe should be called out explicitly?